pkg/chain/celo/gen/abi/deposit/Deposit.go

```diff
@@ -0,0 +1,1137 @@
+// Code generated - DO NOT EDIT.
+// This file is a generated binding and any manual changes will be lost.
+
+package abi
+
+import (
+	"math/big"
+	"strings"
+
+	ethereum "github.com/celo-org/celo-blockchain"
```

Comment:
Interesting... is it something that wasn't renamed?

Reply:
That's done by the Celo `abigen`. This is because Celo team renamed the package `celo-org/celo-blockchain` from `ethereum` to `celo` but they still alias them as `ethereum` to integrate upstream changes in an easy way. Details here: https://github.com/celo-org/celo-blockchain/pull/1375#discussion_r574786037